template/src/pageComponents/buy-me-coffee/Introduction.tsx

```diff
@@ -0,0 +1,27 @@
+import Image from 'next/image';
+import UserAvatar from './Avatar.svg';
+
+export default function Introduction() {
+  const coffeeReceivedCount = 90;
+  const userName = '0xPudgy';
+  const aboutUser = 'I am a content creator producing videos about NFTs!';
```

Comment:
Should this variable be dynamic in any way?

Reply:
Yes this variables should be dynamic. However, I am not sure about the hook from which to retrieve these variables.